components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -133,12 +136,115 @@ func (g *graphqlizer) KymaConfigToGraphQL(in gqlschema.KymaConfigInput) (string,
 	return g.genericToGraphQL(in, `{
 		{{- if .Version }}
 		version: "{{.Version}}"
-		{{- end }}
-		{{- if .Modules }}
-		modules: [
-			{{- range $i, $e := .Modules }}
-			{{- if $i}}, {{- end}} {{ $e }}
-			{{- end }}]
+        components: [
```

Comment:
this is a temporary solution - the contract has changed and we need to send components instead of modules. The task about optional components will change this part